integration/cluster.go

```diff
@@ -80,7 +88,7 @@ func (c *cluster) fillClusterForMembers() error {
 	addrs := make([]string, 0)
 	for _, m := range c.Members {
 		scheme := "http"
-		if !m.PeerTLSInfo.Empty() {
+		if m.PeerTLSInfo != nil {
```

Comment:
Right, there are some places where the nil gets replaced with an empty tls so bits that expect it are happy. The change was primarily meant to sanely rip out the goofy `UsePeerTLS` flag via nil pointers.